### PR TITLE
docs: add "variant" prompt and description to custom plugin development docs

### PR DESCRIPTION
**Pull request by kgpayne**
Related to [sdk#1241](https://github.com/meltano/sdk/pull/1241)

### CODE REVIEW

File: docs/src/_tutorials/custom-extractor.md [hunk 1]
```diff
@@ -326,9 +327,9 @@ Run the command below to add the extractor as a custom extractor not hosted on M
 meltano add --custom extractor tap-jsonplaceholder
 ```
 
-You will be prompted to input the namespace URL. Choose ```tap-jsonplaceholder```.
+You will be prompted to input the namespace URL. Choose `tap-jsonplaceholder`.
 
-Also, choose ```-e …/tap-jsonplaceholder``` as the pip_url since you are working with a local extractor project. This should be the full path to your custom extractor.
+Also, choose `-e …/tap-jsonplaceholder` as the pip_url since you are working with a local extractor project. This should be the full path to your custom extractor.
```
Comment (WillDaSilva) [inline docs/src/_tutorials/custom-extractor.md:332]: I don't think the `…` character should be used here.

File: docs/src/_tutorials/custom-extractor.md [hunk 2]
```diff
@@ -209,12 +210,11 @@ class CommentsStream(jsonplaceholderStream):
     ).to_dict()
 ```
 
-The ```tap.py``` file defines the tap settings and the available streams, which is the comments stream in this case. You can find the available stream types in the STREAM_TYPES array.
-
-The ```streams.py``` file configures the comments stream to use the /comments path and also sets the properties of the extracted fields.
+The `tap.py` file defines the tap settings and the available streams, which is the comments stream in this case. You can find the available stream types in the STREAM_TYPES array.
 
-Finally, change the url_base in the ```tap-jsonplaceholder/tap_jsonplaceholder/client.py``` file to https://jsonplaceholder.typicode.com.
+The `streams.py` file configures the comments stream to use the /comments path and also sets the properties of the extracted fields.
```
Comment (WillDaSilva) [inline docs/src/_tutorials/custom-extractor.md:215]:
```suggestion
The `streams.py` file configures the comments stream to use the `/comments` path and also sets the properties of the extracted fields.
```

File: docs/src/_tutorials/custom-extractor.md [hunk 3]
```diff
@@ -75,9 +74,10 @@ cookiecutter https://github.com/meltano/sdk --directory="cookiecutter/tap-templa
 
 After running the above command, you will be prompted to configure your project.
 
-- Type `jsonplaceholder` as your source name
+- Type `jsonplaceholder` as your source name.
 - Then input your first name and last name.
 - You can leave the tap_id and library name as the default suggested names.
+- If you are planning to distribute your project on PyPI and the tap_id is already in use, you can set a variant name. e.g. 'meltanolabs' or 'pipelinewise'.
```
Comment (WillDaSilva) [inline docs/src/_tutorials/custom-extractor.md:80]:
```suggestion
- You can leave the `tap_id` and library name as the default suggested names.
- If you are planning to distribute your project on PyPI and the `tap_id` is already in use, you can set a variant name. e.g. 'meltanolabs' or 'pipelinewise'.
```

File: docs/src/_tutorials/custom-extractor.md [hunk 4]
```diff
@@ -209,12 +210,11 @@ class CommentsStream(jsonplaceholderStream):
...
 
-The ```tap.py``` file defines the tap settings and the available streams, which is the comments stream in this case. You can find the available stream types in the STREAM_TYPES array.
-
-The ```streams.py``` file configures the comments stream to use the /comments path and also sets the properties of the extracted fields.
+The `tap.py` file defines the tap settings and the available streams, which is the comments stream in this case. You can find the available stream types in the STREAM_TYPES array.
 
-Finally, change the url_base in the ```tap-jsonplaceholder/tap_jsonplaceholder/client.py``` file to https://jsonplaceholder.typicode.com.
+The `streams.py` file configures the comments stream to use the /comments path and also sets the properties of the extracted fields.
 
+Finally, change the url_base in the `tap-jsonplaceholder/tap_jsonplaceholder/client.py` file to https://jsonplaceholder.typicode.com.
```
Comment (edgarrmondragon) [inline docs/src/_tutorials/custom-extractor.md:217]:
```suggestion
Finally, change the value of `url_base` in the `tap-jsonplaceholder/tap_jsonplaceholder/client.py` file to https://jsonplaceholder.typicode.com.
```

**Review by tayloramurphy — APPROVED** [review]

**Review by edgarrmondragon — APPROVED** [review]

**Pull request merged**